package.json

```diff
@@ -270,7 +270,6 @@
         "msw": "^0.49.0",
         "path-browserify": "^1.0.1",
         "pixelmatch": "^5.3.0",
-        "playwright-core": "1.29.2",
```

Comment:
Wasn't sure if this was needed but tried removing it and nothing broke. Basing things off [this release note](https://github.com/microsoft/playwright/releases/tag/v1.35.0) it suggested relying on `@playwright/test` for types